src/core/main.c

```diff
@@ -110,7 +117,12 @@ main(int argc, char** argv)
   if (sizeof(JsRef) != sizeof(int)) {
     FATAL_ERROR("JsRef doesn't have the same size as int.");
   }
+  emscripten_exit_with_live_runtime();
+  return 0;
+}
 
+int
+pyodide_init(void){
```

Comment:
Looks like `clang-format` is complaining about this line?

```sh
clang-format-6.0 src/core/main.c | diff src/core/main.c -
```

Reply:
Thanks!